specification/fluidrelay/resource-manager/Microsoft.FluidRelay/preview/2021-08-30-preview/fluidrelay.json

```diff
@@ -0,0 +1,919 @@
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.FluidRelay/fluidRelayServers": {
+      "get": {
+        "operationId": "FluidRelayServers_ListBySubscription",
+        "summary": "List all Fluid Relay servers in a subscription.",
+        "x-ms-examples": {
+          "List all Fluid Relay servers in a subscription": {
+            "$ref": "./examples/FluidRelayServers_ListBySubscription.json"
+          }
+        },
+        "tags": [
+          "FluidRelayServers"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/FluidRelayServerList"
+            },
+            "description": "Paged response"
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": "Error in operation."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.FluidRelay/fluidRelayServers": {
+      "get": {
+        "operationId": "FluidRelayServers_ListByResourceGroup",
+        "tags": [
+          "FluidRelayServers"
+        ],
+        "summary": "List all Fluid Relay servers in a resource group.",
+        "x-ms-examples": {
+          "List all Fluid Relay servers in a resource group": {
+            "$ref": "./examples/FluidRelayServers_ListByResourceGroup.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/FluidRelayServerList"
+            },
+            "description": "Paged response"
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": "Error in operation."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.FluidRelay/fluidRelayServers/{fluidRelayServerName}/fluidRelayContainers/{fluidRelayContainerName}": {
```

Comment:
>fluidRelayContainerName [](http://example.com/codeflow?start=162&length=23)

Is this a tracked or proxy resource? And if user can Delete it what creates it and when? Why no PUT or PATCH?